### PR TITLE
Fix/viva ms create case new data struct

### PR DESCRIPTION
**Pull request by dannil76**
Implementerat ny datastruktur för ett case
Se mer info här: https://app.clickup.com/t/e0yamz

### CODE REVIEW

**Comment by helsingborgsstad** [comment]
Task linked: [CU-e0yamz createVivaCase lambda ska stödja ny datastruktur för case data (DynamoDB)](https://app.clickup.com/t/e0yamz)

**Review by jwinding — APPROVED** [review]

File: services/viva-ms/lambdas/createVivaCase.js [hunk 1]
```diff
@@ -110,20 +121,14 @@ async function putRecurringVivaCase(PK, workflowId, period) {
...
       },
-      answers: [],
-      currentPosition: {
-        currentMainStep: 1,
-        currentMainStepIndex: 0,
-        index: 0,
-        level: 0,
-      },
+      activeFormId: initialRecurringForm.id,
+      forms: [initialRecurringForm],
```
Comment (TeddyAndersson) [inline services/viva-ms/lambdas/createVivaCase.js:131]: Was this not supposed to be an object with the id as a key?

**Review by TeddyAndersson — COMMENTED** [review]
Comment on the data structure.

File: services/viva-ms/lambdas/createVivaCase.js (hunk 1 above)
Comment (dannil76) [reply services/viva-ms/lambdas/createVivaCase.js:131]: Hmm...

Comment (dannil76) [reply services/viva-ms/lambdas/createVivaCase.js]: Ska nu vara löst. Kom på ärendet initialt bör innehålla alla relevant formulär. Dvs löpande samt stickprov.
Id'n till formulären sparas lämpligen under ssm params

**Review by jwinding — APPROVED** [review]
Was too quick with my last approval, hopefully I've taken my time now, but I think it looks good, data structure looks correct.

**Review by TeddyAndersson — APPROVED** [review]
I Approve these changes

**Pull request merged**